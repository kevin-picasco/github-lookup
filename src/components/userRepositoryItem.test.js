import React from 'react';
import { mount } from 'enzyme';
import UserRepositoryItem from "./userRepositoryItem";

describe('userRepositoryItem', () => {
    it('render <UserRepositoryItem> correctly', () => {
        const mockData = {
            id: 218454167,
            name: 'github-lookup',
            created_at: '2019-10-30T06:01:25Z'
        };

        const component = mount(<UserRepositoryItem userRepository={mockData} />);

        expect(component).toMatchSnapshot();
    });
})