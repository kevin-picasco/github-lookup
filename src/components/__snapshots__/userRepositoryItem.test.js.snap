// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userRepositoryItem render <UserRepositoryItem> correctly 1`] = `ReactWrapper {}`;
